src/Nest/CommonOptions/TimeUnit/Time.cs

```diff
@@ -16,57 +16,99 @@ public class Time : IComparable<Time>, IEquatable<Time>
 		private const double MillisecondsInAnHour = MillisecondsInAMinute * 60;
 		private const double MillisecondsInAMinute = MillisecondsInASecond * 60;
 		private const double MillisecondsInASecond = 1000;
+		private const double NanosecondsInAMillisecond = 100;
```

Comment:
1,000,000 nanoseconds in a millsecond. I think this should be `NanosecondsInATick`